uiowa/uiowa
# Feature: Default icons for person list and pages

Resolves https://github.com/uiowa/uiowa/issues/3726 and resolves https://github.com/uiowa/uiowa/issues/3701. 

- Contains a fix for missing click-target class
- Updates all person lists and full node view to use a default image if no image is selected

# How to test
`yarn workspace uids_base gulp --development`
- As a V2 or V3 site editor I can add a person content type without an image and expect a default icon based image to be used on the node view, people block list view, and the sitenow people page view.   
- As a site editor, I can adjust the image crop and size using the People list block and see the default icon image respond to each setting. 
- As a site editor, I can click on the entire person list card when viewing a page built using the person list block.
- Images should match the screenshots listed on https://github.com/uiowa/uiowa/issues/3701#issuecomment-871722653.
